packages/trade/exchange/src/pods/transfer/transfer.entity.ts

```diff
@@ -61,4 +61,9 @@ export class Transfer extends ExtendedBaseEntity {
     @Column('simple-json', { nullable: true })
     @IsOptional()
     claimData?: IClaimData;
+
+    @ApiProperty({ type: String, required: false })
+    @Column({ nullable: true })
+    @IsOptional()
+    to?: string;
```

Comment:
@JosephBagaric should I add this column or I am supposed to use the `address` instead?

Reply:
Good question. You should use the `address` field instead.

Reply:
Unfortunately I was unable to do so, because in `address` we store the user's exchange deposit address and it is always truthy. 
Changing `to` to `address` also broke the claiming from the exchange for users without organization deposit address.
So I am thinking of simply renaming the columns to match their purpose:
`address` -> `exchangeAddress`
`to` -> `blockchainAddress`
What do you think @JosephBagaric ?

Reply:
well, changing `address` to `exchangeAddress` can also unexpectedly brake several things. 
so I will just rename `to` to `claimAddress` and that's it.